src/EbmlMaster.cpp

```diff
@@ -243,8 +243,7 @@ std::vector<std::string> EbmlMaster::FindAllMissingElements()
       EbmlMaster *childMaster = (EbmlMaster *)childElement;
 
       std::vector<std::string> childMissingElements = childMaster->FindAllMissingElements();
-      for (size_t s = 0; s < childMissingElements.size(); s++)
-        missingElements.push_back(childMissingElements[s]);
+      std::copy(childMissingElements.begin(), childMissingElements.end(), missingElements.begin());
```

Comment:
Uhm, this isn't really the same behavior. The elements from `childMissingElements` should be appended to `missingElements`, so at least there should be a `std::back_inserter(missingElements)` or something like that.

As is this change is most likely wrong.